.circleci/config.yml

```diff
@@ -150,14 +152,7 @@ jobs:
           name: Run testnet
           command: |
             cd lunie-backend
-            npm run start -- -d
-      - run:
-          # needs to be done in CI to be sure it is build once we start the tests
-          name: Build for E2E test
-          command: npm run test:e2e:build
-          environment:
-            VUE_APP_GRAPHQL_URL: http://127.0.0.1:4000 # Browserstack tunneling doesn't work with "localhost"
-            NETWORK: local-cosmos-hub-testnet # Connect directly to the testnet
+            docker-compose up --build -d
```

Comment:
What is `-d`? Daemon? I can't find it in the docs.

I feel the long form 'wordy' version is better for configs.

Reply:
done